packages/textarea/package.json

```
@@ -18,7 +18,8 @@
   "dependencies": {
     "@smashing/text-input": "^0.3.0",
     "@smashing/theme": "^0.13.0",
-    "@smashing/typography": "^0.4.28"
+    "@smashing/typography": "^0.4.28",
+    "react-custom-scrollbars": "^4.2.1"
```

Lock this dependency
```suggestion
    "react-custom-scrollbars": "4.2.1"
```